thirdeye/thirdeye-frontend/app/mirage/fixtures/applicationAnomalies.js

```diff
@@ -18,10 +19,11 @@ export default [
     baseline: 1000,
     feedback: 'Yes, but New Trend',
     metricName: 'Metric Name 1',
-    metricId: 12345,
+    metricanomalyId: 12345,
```

Comment:
sanity check: is this the format the back end will send this? no camelCase? `metricAnomalyId`

Reply:
Ad darn. I did a replace all for `Id:`..with `anomalyId`. Need to revert.

Reply:
DONE